src/qml/bitcoin.cpp

```diff
@@ -90,6 +90,16 @@ void DebugMessageHandler(QtMsgType type, const QMessageLogContext& context, cons
         LogPrintf("GUI: %s\n", msg.toStdString());
     }
 }
+
+bool SettingsFileExists(ArgsManager& argsman)
```

Comment:
We also wouldn't show onboarding if a bitcoin.conf exists within the datadir, as that's another way to provide a configuration